pages/dette-har-vi-gjort/index.tsx

```diff
@@ -4,22 +4,50 @@ import { GetStaticProps } from "next"
 import Layout from "../../components/Layout"
 import ProjectCard from "../../components/ProjectCard"
 import HighlightedProjectsCarousel from "../../components/HighlightedProjectsCarousel"
+import { useState } from "react"
+import Button from "../../components/buttons/Button"
 
-type ProjectsType = {
+type projectPageProps= {
```

Comment:
```suggestion
type ProjectsProps= {
```

Det er nice å alltid ha PascalCase på types, og at typenavnet til propsene heter NavnPåComponentProps, få ProjectsProps i dette tilfellet